src/App.js

```diff
@@ -117,14 +136,14 @@ const App = ({ stateCode, correctionsMonthlyData, correctionsAnnualData }) => {
     sourceData: monthlySourceData,
   };
 
-  const annualPopulationsChartData = generateChartData(
+  const annualPopulationsChartData = generateCorrectionsChartData(
     annualStateMetricData,
     [POPULATION_PRISON, POPULATION_PAROLE, POPULATION_PROBATION],
     ["Prison Population", "Parole Population", "Probation Population"],
     true
   );
 
-  const annualPrisonAdmissionsChartData = generateChartData(
+  const annualPrisonAdmissionsChartData = generateCorrectionsChartData(
     annualStateMetricData,
     [ADMISSIONS, ADMISSIONS_NEW_COMMITMENTS, ADMISSIONS_FROM_PAROLE, ADMISSIONS_FROM_PROBATION],
     [
```

Comment:
Post-Release Supervision (just below)

Reply:
Releases to Parole also change to Releases to Post-Release Supervision?

Reply:
fixed